client/css/style.css

```diff
@@ -374,8 +370,7 @@ kbd {
 }
 
 #viewport.rt #chat .sidebar {
-	-webkit-transform: translate3d(180px, 0, 0);
-	transform: translate3d(180px, 0, 0);
+	right: -180px;
```

Comment:
You are correct that it would be doing a repaint, however this was needed to fix the font rendering issue.